WorkflowUI/Sources/Observation/WorkflowUIViewController.swift

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2023 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#if canImport(UIKit)
+import Foundation
+import UIKit
+
+/// Ancestor type from which all ViewControllers in WorkflowUI inherit.
+open class WorkflowUIViewController: UIViewController {
```

Comment:
cc @n8chur i know you've got a feature branch that this might conflict with. any issues with adding this new common supertype you can think of?